### PR TITLE
Fix sqlite3 build on pipeline

### PR DESCRIPTION
**Pull request by ultmaster**
To fix sqlite3 at 5.0.0 because binary packages are missing for new versions. I don't know why such change works.

Also deleted unused pipeline files.

Another possible solution is to upgrade node-gyp version. It works as tested in #3296.

### CODE REVIEW

**Review by liuzhe-lz — APPROVED** [review]

**Comment by ultmaster** [comment]
Seems that part of this PR is already done in #3288. Pending for merge.

**Comment by liuzhe-lz** [comment]
> 
> 
> Seems that part of this PR is already done in #3288. Pending for merge.

I updated that PR to prevent conflict. Suggest to merge this one also.

**Review by liuzhe-lz — APPROVED** [review]

**Review by QuanluZhang — APPROVED** [review]

File: ts/nni_manager/package.json [hunk 1]
```diff
@@ -21,7 +21,7 @@
     "lockfile": "^1.0.4",
     "python-shell": "^2.0.1",
     "rx": "^4.1.0",
-    "sqlite3": "^5.0.0",
+    "sqlite3": "5.0.0",
```
Comment (SparkSnail) [inline ts/nni_manager/package.json:24]: Use a fixed version for sqlite3? then suggest to add annotation here to explain the reason.

Comment (ultmaster) [reply ts/nni_manager/package.json:24]: Already added in description of this PR. Is there anyway to comment a JSON file?

**Review by SparkSnail — APPROVED** [review]

File: ts/nni_manager/package.json (hunk 1 above)
Comment (SparkSnail) [reply ts/nni_manager/package.json:24]: Ok, seems it hack for adding comments in this file.

**Pull request merged**